2_process/src/process_data.R

```diff
@@ -1,20 +1,12 @@
-
-library(dplyr)
-library(readr)
-library(stringr)
-library(sbtools)
-library(whisker)
-
 # Read in the data downloaded in the prior step
-read_eval_data = function() {
-  mendota_file = "1_fetch/out/model_RMSes.csv"
+read_eval_data = function(mendota_file = "1_fetch/out/model_RMSes.csv") {
   eval_data <- readr::read_csv(mendota_file, col_types = 'iccd')
   return(eval_data)
 }
 
 # Process the data
-process_data <- function() {
-  eval_data <- read_eval_data() %>%
+process_data <- function(mendota_file = "1_fetch/out/model_RMSes.csv") {
```

Comment:
Again, for best practices, although `targets` knows this is a file target and you are returning the filename at the end of the function, go ahead and include that `output_file` parameter as an argument to the function, so that we can see what the filename is right in the `_targets.R` file, and so that it's not hard-coded within the function itself.

Reply:
Done!